### DISCUSSION TITLE
Can we assume a particular order from a search over a NativeStore?

### DISCUSSION THREAD

**Discussion by ate47** [eclipse-rdf4j/rdf4j#4294] [Q&A] [answered]
Hi!

We are using results from a NativeStore repository connection and we want to use this kind of results without having to order them by ourself.

My question is, when we do a ``getStatements(null, null, null, false)``, can we assume a particular order from the triples of the result iterator or not?

Antoine

**Comment by hmottestad** [comment, marked as answer]
You can not assume any particular order.

**Reply by ate47** [reply]
Ok, thank you!
